### PR TITLE
Src 6629 Update sr config demos

### PR DESCRIPTION
**Pull request by georgiablanco**
## Proposed changes

Adding sr-config demos to their own sr_demo folder and refactoring so they work with unimanual/bimanual, left/right, tactile/non-tactile, SIM/NON-SIM

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added automated tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [X] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [X] Manually tested on hardware (if hardware specific or related).

## Documentation

- [X] No documentation required to be added. *Documentation will be added at the end of the week*
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: sr_demos/package.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>sr_demos</name>
+  <version>0.0.0</version>
+  <description>The sr_demos package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="user@todo.todo">user</maintainer>
```
Comment (BenStarmerSmith) [inline sr_demos/package.xml]: Does it matter that this hasn't been set?

File: sr_demos/scripts/sr_demos/close.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                        'LFJ5': 0.0,
+                        'WRJ1': 0.0, 'WRJ2': 0.0}
+    close_thumb_no_id = {'THJ1': 0.52, 'THJ2': 0.61, 'THJ3': 0.0, 'THJ4': 1.20, 'THJ5': 0.17}
+    # sr-config one
+    # close_hand_no_id = {'FFJ1': 1.5707, 'FFJ2': 1.5707, 'FFJ3': 1.5707, 'FFJ4': 0.0,
+    #                       'MFJ1': 1.5707, 'MFJ2': 1.5707, 'MFJ3': 1.5707, 'MFJ4': 0.0,
+    #                       'RFJ1': 1.5707, 'RFJ2': 1.5707, 'RFJ3': 1.5707, 'RFJ4': 0.0,
+    #                       'LFJ1': 1.5707, 'LFJ2': 1.5707, 'LFJ3': 1.5707, 'LFJ4': 0.0, 'LFJ5': 0.0,
+    #                       'THJ1': 0.75, 'THJ2': 0.45, 'THJ3': 0.0, 'THJ4': 1.2, 'THJ5': 0.35,
+    #                       'WRJ1': 0.0, 'WRJ2': 0.0}
```
Comment (BenStarmerSmith) [inline sr_demos/scripts/sr_demos/close.py]: Is all this dict going to be used later on down the line?

File: sr_demos/scripts/sr_demos/open.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    hand_commander = SrHandCommander(name=hand_name)
+
+    open_hand_no_id = {'FFJ1': 0.0, 'FFJ2': 0.0, 'FFJ3': 0.0, 'FFJ4': 0.0,
+                       'MFJ1': 0.0, 'MFJ2': 0.0, 'MFJ3': 0.0, 'MFJ4': 0.0,
+                       'RFJ1': 0.0, 'RFJ2': 0.0, 'RFJ3': 0.0, 'RFJ4': 0.0,
+                       'LFJ1': 0.0, 'LFJ2': 0.0, 'LFJ3': 0.0, 'LFJ4': 0.0, 'LFJ5': 0.0,
+                       'THJ1': 0.0, 'THJ2': 0.0, 'THJ3': 0.0, 'THJ4': 0.0, 'THJ5': 0.0,
+                       'WRJ1': 0.0, 'WRJ2': 0.0}
+
+    # NOT SURE WHY THIS IS INCLUDED - CAN I REMOVE? #####
```
Comment (BenStarmerSmith) [inline sr_demos/scripts/sr_demos/open.py]: Agree on this point, maybe just remove if its been commented out for awhile?

File: sr_demos/package.xml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>sr_demos</name>
+  <version>0.0.0</version>
+  <description>The sr_demos package</description>
+  <maintainer email="software@shadowrobot.com">Shadow Robot's software team</maintainer>
+
+  <license>GPL</license>
+
+  <url type="website">http://ros.org/wiki/sr_example</url>
```
Comment (mykolasjuraitis) [inline sr_demos/package.xml]: This is probable not needed?

**Review by mykolasjuraitis — APPROVED** [review]

**Review by BenStarmerSmith — APPROVED** [review]

**Pull request merged**